searchlib/src/vespa/searchlib/attribute/i_enum_store_dictionary.h

```diff
@@ -52,6 +52,7 @@ class IEnumStoreDictionary : public vespalib::datastore::IUniqueStoreDictionary
     virtual Index remap_index(Index idx) = 0;
     virtual void clear_all_posting_lists(std::function<void(EntryRef)> clearer) = 0;
     virtual void update_posting_list(Index idx, const vespalib::datastore::EntryComparator& cmp, std::function<EntryRef(EntryRef)> updater) = 0;
+    virtual bool check_posting_lists(std::function<EntryRef(EntryRef)> updater) = 0;
```

Comment:
Consider adding a comment. It is not clear what this checking implies and what the updater is supposed to do.